### PR TITLE
Implement trigger updated option3

### PR DESCRIPTION
**Pull request by Mnakagawa**
Issue #1452

### CODE REVIEW

File: server/hap2/hatohol/hap2_nagios_ndoutils.py [hunk 1]
```diff
@@ -117,6 +118,11 @@ def __parse_url(self, url):
 
         return server, port, database
 
+    def __convert_to_nagios_time(self, hatohol_time):
+        over_second = hatohol_time.split(".")[0]
+        nag_datetime = datetime.datetime.strptime(over_second, '%Y%m%d%H%M%S')
```
Comment (cosmo0920) [inline server/hap2/hatohol/hap2_nagios_ndoutils.py]: Please avoid unnecessary variable abbreviation If you can.
It makes hard to understand what variables mean.

**Comment by cosmo0920** [comment]
Oh, Too late to add comments.... :cry:

File: server/hap2/hatohol/hap2_nagios_ndoutils.py (hunk 1 above)
Comment (Mnakagawa) [inline server/hap2/hatohol/hap2_nagios_ndoutils.py]: I'm sorry...
I will take care from the next time.

Comment (cosmo0920) [inline server/hap2/hatohol/hap2_nagios_ndoutils.py]: Well, don't mind. 
Above commend is just an idea. :-p

**Pull request merged**